src/main/java/uk/gov/hmcts/reform/notificationservice/model/out/NotificationsResponse.java

```diff
@@ -34,7 +35,9 @@
     public final List<NotificationInfo> notifications;
 
     public NotificationsResponse(List<NotificationInfo> notifications) {
-        this.notifications = notifications;
+        this.notifications = notifications.stream()
+            .sorted(Comparator.comparingLong(i -> Long.parseLong(i.id)))
+            .collect(Collectors.toList());
```

Comment:
static import

Reply:
Please elaborate on your comment?

Reply:
you can import `Collectors` static  and in code you can use `toList()` instead  of `Collectors.toList()`